llvm/test/Transforms/SLPVectorizer/RISCV/revec-getGatherCost.ll

```diff
@@ -0,0 +1,31 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -mtriple=riscv64 -mcpu=sifive-x280 -passes=slp-vectorizer -S -slp-revec -slp-max-reg-size=1024 -slp-threshold=-10 %s | FileCheck %s
```

Comment:
Would be good to add remarks check to see cost changes

Reply:
done